bot/cogs/utils.py

```diff
@@ -230,7 +230,16 @@ def check(m: Message) -> bool:
             await ctx.send(embed=embed)
             return
 
-        # handle if it's a search string
+        # Try to handle first exact word due difflib.SequenceMatched may use some other similar word instead
+        # exact word.
+        for i, line in enumerate(zen_lines):
+            if search_value.lower() in line.lower():
+                embed.title += f" (line {i}):"
+                embed.description = line
+                await ctx.send(embed=embed)
+                return
+
+        # handle if it's a search string and not exact word
```

Comment:
While this solution works for this particular case, it now picks the first line that contains the word as a substring. For example, if you swap the lines containing explicit and explicitly around, it will match the wrong one again. @sco1 has an alternative suggestion that matches based on words instead of an entire sentence to truly prefer full word matches.